GEOSCHEMchem_GridComp/GEOSCHEMchem_GridCompMod.F90

```diff
@@ -1992,7 +1993,7 @@ SUBROUTINE Initialize_( GC, Import, Export, Clock, RC )
 #if defined( MODEL_GEOS )
     ! Top stratospheric level
     CALL ESMF_ConfigGetAttribute( GeosCF, value_LLSTRAT,         & 
-                                  Default = 59,                     &
+                                  Default = 72,                     &
```

Comment:
@christophkeller A question. Should this be hardcoded to 72? Or should it be something like LM...or even can it be something like LM (i.e., does it need a `parameter`)? 

If we move to 132 levels, this could be a fun thing to track down.

Reply:
Oh. Hmm. It looks like `GEOSCHEMchem_GridComp/GEOSCHEMchem_GridComp.rc` also has a hard-coded 72. RC files we can transmute at run-time, so that's not as crucial, but I'm always wary of hard-coded 72s in GEOS.

Reply:
Looks like @mmanyin had the same question above I was just too blind to see it.

Reply:
Good point. The hardcoded default value is a relict from older versions of GEOS-Chem where chemistry was restricted to 59 levels. I will change the default to LM, I think that makes a lot of sense.